src/Core/Geometry/TopologicalMesh.cpp

```diff
@@ -493,10 +498,10 @@ void TopologicalMesh::initWithWedge( const TriangleMesh& triMesh ) {
             else
             { vh = vtr->second; }
 
-            face_vhandles[j]    = vh;
-            face_normals[j]     = n;
-            face_vertexIndex[j] = inMeshVertexIndex;
-            face_wedges[j]      = WedgeIndex {int( i )};
+            face_vhandles[j] = vh;
+            face_normals[j]  = n;
+            face_wedges[j] =
+                WedgeIndex {static_cast<WedgeIndex::Index::IntegerType>( inMeshVertexIndex )};
```

Comment:
Do you really need a static cast ? Can’t we call the constructor directly with the integer as input?

Reply:
it's an unsigned int, so
```
warning: narrowing conversion of ‘inMeshVertexIndex’ from ‘unsigned int’ to ‘Ra::Core::Utils::Index::IntegerType’ {aka ‘int’}
```

Reply:
We could also add an implicit cast from unsigned int to Index

Reply:
Yes, it would make sens to add this constructor to the Index class

Reply:
Are you wiling to do it in this PR ?

Reply:
Nope ;) feel free to add it